### ISSUE TITLE
[Feature Request] Spatie Medialibrary Upload and Upload Multiple Field

### ISSUE THREAD

**Issue by ziming** [Laravel-Backpack/CRUD#4497]
# Feature Request

### What's the feature you think Backpack should have?

Upload and Upload Multiple Field but work with spatie media library

### Have you already implemented a prototype solution, for your own project?

Hmm only for the backend part for single file collection, but not the blade field views. I think you can see mine in the gitter chat logs.

### Do you see this as a core feature or an add-on?

Core. Spatie MediaLibrary is pretty much the defacto library for managing file uploads in laravel projects now.

**Comment by tabacitu** [comment]
Yup, it's on the roadmap, probably going to do it by end of August 2022 🎉
